java/java.lsp.server/vscode/package.json

```diff
@@ -84,6 +84,11 @@
 					"type": "boolean",
 					"default": false,
 					"description": "Enable Run/Debug test in editor"
+				},
+				"netbeans.groovySupport.enabled": {
```

Comment:
Does changing the value of this option automatically restarts the JVM or does it have to be restarted manually?

Reply:
Not yet.  Should we ever fix it / document it, if we believe we can improve things during 12.5 release ?